### DISCUSSION TITLE
NiceGUI support multi language like golang, rust, and other

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#781] [Ideas / Feature Requests]
Is it possible that in the future NiceGUI will support multi devices and multi languages. And is there such a plan . If not, no problem. I just want to know thanks

**Comment by rodja** [comment]
Each new language would be an enormous task in itself. And some languages may not support the concepts which makes NiceGUI strong. But we are open for any contributions. Maybe someone wants to start a spike project for a specific language?

**Comment by bobwatcherx** [comment]
if it exists . flet also does the same thing so it's not only python but
can golang .net javascript . I will follow the progress
